tests/regression/array/CMakeLists.txt

```diff
@@ -5,7 +5,13 @@ target_link_libraries(gen_regression_array_driver_fortran gen_regression_array_f
 add_test(NAME gen_regression_array_driver_fortran
     COMMAND $<TARGET_FILE:gen_regression_array_driver_fortran>)
 
-if(CUDA_AVAILABLE AND "${CMAKE_CXX_COMPILER_VERSION}" VERSION_GREATER_EQUAL "6")
```

Comment:
What was your idea with the original condition? I guess selecting gnu >=6?

Reply:
Yes :)